web/api/v1/api_test.go

```diff
@@ -128,15 +128,15 @@ var (
 	scrapeStart = time.Now().Add(-11 * time.Second)
 )
 
-func (t testTargetRetriever) TargetsActive() map[string][]*scrape.Target {
+func (t *testTargetRetriever) TargetsActive() map[string][]*scrape.Target {
```

Comment:
Why does this need a pointer receiver?

Reply:
You're 100% correct - I don't need it for Active/Dropped. Thanks!